frontend/src/helpers/pagination/get-valid-page-from-search-params/get-valid-page-from-search-params.helper.ts

```diff
@@ -0,0 +1,10 @@
+import { PaginationDefaultValue } from 'common/enums/enums';
+
+const getValidPageFromSearchParams = (pageFromQuery: number): number => {
+  const page =
+    pageFromQuery > 0 ? pageFromQuery : PaginationDefaultValue.DEFAULT_PAGE;
```

Comment:
```suggestion
  const page =  pageFromQuery > 0 
    ? pageFromQuery 
    : PaginationDefaultValue.DEFAULT_PAGE;
```

Reply:
Prettier auto-format  :) And when changing it, git doesn't consider it as a change, since nothing is changed there